app/api/currentuser/get_group_invitations.feature

```diff
@@ -53,7 +53,7 @@ Feature: Get group invitations for the current user
           "description": "Group for our friends",
           "type": "Friends"
         },
-        "status_date": "{groups_groups[4][sStatusDate]}",
+        "status_date": "{{db("groups_groups[4][sStatusDate]")}}",
```

Comment:
Checking that the status_date is what is been actually returned does not help much in terms of testing (a bit like testing that `sum(1,2)` is equal to`2+1` instead of checking the static value `3`). As it is bound to `now()` by the db, I assume you cannot really force and get the actual date without annoying changes... but wouldn't it be better to kind of test the value itself, so checking for instance that it is equal to now()+-1sec or something similar?

Reply:
The db() function doesn't read from the DB. It returns data from DataTables. For instance, here it returns relativeTime(-166h).